app/views/merge_requests/organisations.html.erb

```diff
@@ -0,0 +1,54 @@
+<% content_for :before_content do %>
+
+  <% title = "Tell us if your organisation is merging" %>
+  <% content_for :title, title %>
+  <%= govuk_back_link href: merge_request_organisation_path(id: @merge_request.requesting_organisation_id) %>
+<% end %>
+
+<h2 class="govuk-heading-l">Which organisations are merging?</h2>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+   <p class="govuk-body">Add all organisations to be merged - we have already added your own.</p>
+
+<%= form_with model: @merge_request, url: organisations_merge_request_path, method: :patch do |f| %>
+  <%= f.govuk_error_summary %>
+  <p class="govuk-body">Start typing to search</p>
+  <%= render partial: "organisation_relationships/related_organisation_select_question", locals: {
+    field: :merging_organisation,
+    question: Form::Question.new("", { "answer_options" => @answer_options }, nil),
+    f:,
+  } %>
+  <%= f.govuk_submit "Add organisation", classes: "govuk-button--secondary" %>
+  <%= govuk_table do |table| %>
+    <%= table.body do |body| %>
+        <%= body.row do |row| %>
+            <% row.cell(text: @merge_request.requesting_organisation.name) %>
+            <% row.cell(html_attributes: {
+                scope: "row",
+                class: "govuk-!-text-align-right",
+                }) %>
+        <% end %>
+    <% end %>
+  <% @merge_request.merging_organisations&.each do |merging_organisation| %>
```

Comment:
minor thing, shouldn't need a nil guard as the return will always be a collection